uber/models.py

```diff
@@ -644,12 +661,12 @@ def is_group_leader(self):
         return self.group and self.id == self.group.leader_id
 
     @property
-    def shirt_size_marked(self):
-        return self.shirt not in [NO_SHIRT, SIZE_UNKNOWN]
+    def can_check_in(self):
+        return self.paid != NOT_PAID and self.status in [COMPLETED_STATUS, PRINTED_STATUS] and not self.is_unassigned
```

Comment:
This is something that should be in master, but PRINTED_STATUS is currently AC-only. I may create a config option to handle this instead.